objects/9ad187/script.lua

```diff
@@ -44,6 +44,8 @@ exploratoryVOTD = false
 exploratoryBODAN = false
 exploratoryWar = false
 
+scaleBoard = true
```

Comment:
can you group this variable with the other variant settings?

Reply:
also not too sure about having this enabled by default

Reply:
I think a slightly larger board is a nice quality-of-life improvement that should be on by default. I can't imagine why somebody would want it off, at least with the included layouts.

Reply:
i checked with MJ and they feel it should be disabled by default as well